servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ManagementTests.java

```diff
@@ -372,39 +391,39 @@ public void getTopicAndSubscriptionRuntimeInfoTest() throws ExecutionException,
     public void messagingEntityNotFoundExceptionTest() throws ServiceBusException, InterruptedException, ExecutionException {
         try {
             Utils.completeFuture(this.managementClientAsync.getQueueAsync("NonExistingPath"));
-        } catch (MessagingEntityNotFoundException e) {}
+        } catch (MessagingEntityNotFoundException e) { }
 
         try {
             Utils.completeFuture(this.managementClientAsync.getTopicAsync("NonExistingPath"));
-        } catch (MessagingEntityNotFoundException e) {}
+        } catch (MessagingEntityNotFoundException e) { }
 
         try {
             Utils.completeFuture(this.managementClientAsync.getSubscriptionAsync("NonExistingTopic", "NonExistingPath"));
-        } catch (MessagingEntityNotFoundException e) {}
+        } catch (MessagingEntityNotFoundException e) { }
 
         try {
             Utils.completeFuture(this.managementClientAsync.updateQueueAsync(new QueueDescription("NonExistingPath")));
-        } catch (MessagingEntityNotFoundException e) {}
+        } catch (MessagingEntityNotFoundException e) { }
```

Comment:
I'm surprised there isn't a check style about this not being on a new line and the ones below. If so, I'd fix these in this PR.

Reply:
The checkstyle complains 

  | '{' is not followed by whitespace.
-- | --
 and 

  | '}' is not preceded with whitespace.
-- | --

Reply:
Well, if there are no more checkstyle issues on that line. It's all good. I was confused because it complained about:
```java
Workitem() {}
```
And one of your PRs changed it to:
```java
WorkItem() {
}
```

Reply:
Both solve the CheckStyle error. Which one you would prefer to change to? I can change it to stay consistency

Reply:
No worries. As long as checkstyle issue no longer shows up. :)